tests/suites/0_stateless/03_dml/03_0001_select_aggregator.sql

```diff
@@ -1,5 +1,7 @@
+set enable_planner_v2 = 1;
```

Comment:
I'm not sure if I should enable planner v2 directly in this test suit, or copy these tests to https://github.com/datafuselabs/databend/blob/main/tests/suites/0_stateless/20%2B_others/20_0001_planner_v2.sql 🤔

Reply:
Maybe we should add new test check for planner v2(ignore some errors for merge)?

Reply:
My goal is that each pr can make test cases in a test file or multiple test files pass.

There are two more test files related to aggregation plan, they are blocked by group by and subquery table reference https://github.com/datafuselabs/databend/issues/5140, I am going to use two PRs to solve them. 

So we don't need to ignore errors for the sake of merging?

Reply:
I think directly is ok, thus we can cover old behavior if we enable this setting.

After the new planner is stable, we can remove this setting.

Reply:
I suggest to make a copy of this test for planner v2. As soon as the planner v2 is enabled by default, it will be removed.

Reply:
I thought it again, maybe @leiysky  is right, we should keep both tests stable during refactoring.

So let's make a copy or separate tests like `03_0001_select_aggregator_v2.sql`.

When planner v2 is enabled by default, we could merge the tests into `03_0001_select_aggregator.sql`

Reply:
If we have separate tests, we can remove `set enable_planner_v2 = 0;` at the end of the file

Reply:
Yes, separating tests won't influence old planner development.